management-api-agent-common/src/main/java/com/datastax/mgmtapi/models/RingRange.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.models;
+
+import java.math.BigInteger;
+import java.util.Comparator;
+
+public final class RingRange {
+
+  public static final Comparator<RingRange> START_COMPARATOR
+      = (RingRange o1, RingRange o2) -> o1.start.compareTo(o2.start);
+
+  private final BigInteger start;
+  private final BigInteger end;
+
```

Comment:
Something @olim7t just figured out is that model objects on the Agent side of things likely have to have their attributes as `public` in order for the Object serialization to work. See the code here:
https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/feature/get-tables/management-api-agent-3.x/src/main/java/com/datastax/mgmtapi/rpc/ObjectSerializer3x.java#L61